components/main/MainContent.vue

```diff
@@ -63,3 +68,20 @@ const containerClass = computed(() => {
     </div>
   </div>
 </template>
+
+<style>
+  .beta {
+    width: 56.16px;
+    height: 35.16px;
+    padding: 5px 6px;
+    border-radius: 8px;
+    border: 2px solid var(--c-primary);
+    gap: 17.70120620727539px;
```

Comment:
Where did this value come from? Does it need to be so specific?

Reply:
Oooh I took this giant glob from Figma. I'm realizing we don't even need `gap`. Will remove

Reply:
I'm still laughing at how ridiculous this must be to come across in code

Reply:
Ahh yeah, I thought these looked like Figma styles. Generally, I would recommend not relying on CSS that is generated from tools like Figma. Maybe the generators are better today than they used to be and I'm just being old, but they are usually unreliable, operate in isolation, don't take into account our patterns or best practices and encourage bad habits.

I put together #27 to illustrate how many styles Figma generated that we actually don't need. I'm happy to walk through any of this with you too!